cli/aws_orbit/remote_files/cdk/lambda_sources/cognito_post_authentication/k8s_manage.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import sys
+import subprocess
+from typing import Any, Dict, List, Optional
+
+import boto3
+from kubernetes import config, client
+from kubernetes.client.rest import ApiException
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def handler(event: Dict[str, Any], context: Optional[Dict[str, Any]]) -> Any:
+    user_name = event.get("user_name")
+    user_email = event.get("user_email")
+    user_pool_id = event.get("user_pool_id")
+    expected_user_namespaces = event.get('expected_user_namespaces')
+
+    create_kubeconfig()
+
+    api = client.CoreV1Api()
+
+    manage_user_namespace(kubernetes_api_client=api, expected_user_namespaces=expected_user_namespaces, user_name=user_name, user_email=user_email, user_pool_id=user_pool_id)
+
+def run_command(cmd: str) -> str:
+    """ Module to run shell commands. """
+    cmds = cmd.split(" ")
+    try:
+        output = subprocess.run(cmds, stderr=subprocess.STDOUT, shell=False, timeout=120, universal_newlines=True)
+        print(output)
+    except subprocess.CalledProcessError as exc:
+        # print("Command failed with exit code {}, stderr: {}".format(exc.returncode, exc.output.decode("utf-8")))
+        raise exc
+    return output
+
+def create_kubeconfig() -> bool:
+    KUBECONFIG_PATH = "/tmp/.kubeconfig"
+
+    logger.info(f'Generating kubeconfig in {KUBECONFIG_PATH}')
+    run_command(f"aws eks update-kubeconfig --name orbit-dev-env --role-arn arn:aws:iam::339309931548:role/orbit-dev-env-admin --kubeconfig {KUBECONFIG_PATH}")
+    
+    logger.info("Loading kubeconfig")
+    try:
+        config.load_kube_config(KUBECONFIG_PATH)
+        logger.info("Loaded kubeconfig successfully")
+    except config.ConfigException:
+        raise Exception("Could not configure kubernetes python client")
+
+def manage_user_namespace(kubernetes_api_client: client.CoreV1Api, expected_user_namespaces: List, user_name: str, user_email: str, user_pool_id: str) -> None:
+    api = kubernetes_api_client
+
+    all_ns_raw = api.list_namespace().to_dict()
+    all_ns = [item.get('metadata').get('name') for item in all_ns_raw['items']  if item.get('metadata').get('name').startswith(user_name)]
+
+    # Create user namespace
+    for team, user_ns in expected_user_namespaces.items():
+        if user_ns not in all_ns:
+            logger.info(f'User namespace {user_ns} doesnt exist. Creating...')
+            name = user_ns
+            annotations = {"owner": user_email}
+            labels = {"orbit/user-space": "enabled", "teams": team, "env": "test-env-kf"}
```

Comment:
change "orbit/user-space": "enabled" to  "orbit/space": "team"

Reply:
also add "istio-injection": "enabled"

Reply:
here is how it should look:
  labels:
    orbit/space: user
    orbit/team: lake-user
    orbit/env: test-env-kf
    orbit/user: amirbo
    istio-injection: enabled

notice its not 'teams' or 'env'